AdobeStockAsset/Model/AssetRepository.php

```diff
@@ -86,6 +96,7 @@ public function __construct(
     public function save(AssetInterface $asset): void
     {
         $this->resource->save($asset);
+        $this->saveMultiplyAndAssignToAsset->execute($asset);
```

Comment:
As there is a keyword repository - it should be responsible for saving keywords, all extra entry points to save keyword action should be removed to have a single entry point.

```suggestion
        $this->keywordRepository->save($assetId, $asset->getKeywords());
```